breathecode/authenticate/views.py

```diff
@@ -401,6 +402,11 @@ def put(self, request, new_status=None):
                                                      last_name=item.last_name)
                     profile_academy.save()
 
+                tasks_activity.add_activity.delay(request.user.id,
+                                                  'invite_status_updated',
+                                                  related_type='auth.UserInvite',
+                                                  related_id=item.id)
+
```

Comment:
Use `serializer.is_valid` to make sure this activity will be registered when the data is ok